workshop/4-Messaging/4.2-Braze.ipynb

```diff
@@ -0,0 +1,522 @@
+    "\t<tr>\n",
+    "\t\t<td width=\"115\" class=\"text3\" style=\"color:#212123; font-family:'Noto Sans', Arial,sans-serif; font-size:18px; line-height:22px; text-align:left\">{{result[1].product.name}}</td>\n",
+    "\t\t<td class=\"img\" style=\"font-size:0pt; line-height:0pt; text-align:left\" width=\"10\"></td>\n",
+    "\t\t<td class=\"text4\" style=\"color:#6e6e70; font-family:'Noto Sans', Arial,sans-serif; font-size:12px; line-height:22px; text-align:right\">{{result[1].product.price}}</td>\n",
+    "\t</tr>\n",
+    "</table>\n",
+    "```\n",
+    "\n",
+    "Note the `result[1].product.name` variable will be replaced with the name of the product that is first in the list of products returned, and the same goes for `result[1].product.price` for that product's price.  The same can be done for other product attributes such as the product's image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating a Braze Campaign\n",
+    "\n",
+    "Log in to the Braze [console](https://dashboard.braze.com/auth).  You will need to have a Braze account configured for you to continue this workshop, or use your existing Braze account.\n",
+    "\n",
+    "![Braze Login Screen](images/braze-create-template-0.png)\n",
+    "\n",
+    "On the left hand pane of the Braze console, click the Templates & Media link.  This will show the Templates list screen.  Click the From File button under Basic Email Templates at the top of the screen.\n",
+    "\n",
+    "![](images/braze-create-template-1.png)\n",
+    "\n",
+    "Click the Upload from File button:\n",
+    "\n",
+    "![](images/braze-create-template-2.png)\n",
+    "\n",
+    "Navigate to your local clone of this repository, to the `/workshop/4-Messaging/braze-templates` directory.  \n",
+    "\n",
+    "Select the `braze-connected-content-email-template.zip` file you created earlier and click the Open or Save button to upload the ZIP file to Braze.\n",
+    "\n",
+    "![](images/braze-create-template-3.png)\n",
+    "\n",
+    "Braze will validate the file, and return a status window.  Click the Build Email button to continue.\n",
+    "\n",
+    "![](images/braze-create-template-4.png)\n",
+    "\n",
+    "On the next screen, give the template a name, such as `Braze Connected Content Workshop Template`.\n",
+    "\n",
+    "![](images/braze-create-template-5.png)\n",
+    "\n",
+    "On the bottom right of the screen, click the Save Template button.  \n",
+    "\n",
+    "Make sure to click the Save button often, the editor will not auto-save your changes!\n",
+    "\n",
+    "![](images/braze-create-template-6.png)\n",
+    "\n",
+    "Scroll back up and click the Edit Email Body link\n",
+    "\n",
+    "![](images/braze-create-template-7.png)\n",
+    "\n",
+    "This will take you to the HTML template editor.  \n",
+    "\n",
+    "At the very top, you should see the Connected Content call that calls the Personalize recommendation service.\n",
```

Comment:
The following screenshot shows "<YOUR GLOBAL STORE SERVICE DEPLOYMENT GOES HERE>" rather than the resolved URL replaced in the template above. May be confusing to users.

Reply:
Fixed.